packages/squiggle-lang/__tests__/reducer/various_test.ts

```diff
@@ -127,15 +127,15 @@ describe("stacktraces", () => {
 
     expect(error).toBe(
       `Error: There are function matches for add(), but with different arguments:
-  add(number, number)
```

Comment:
I lost the spacing here and wasn't sure why. Didn't seem like a major issue now, and I want to format better later anyway.